docs/configuration/style-modification.md

```diff
@@ -0,0 +1,16 @@
+# Style Modification
+
+## Overview
+
+This document describes how you can apply the styles for the AsyncApi component.
+
+## Changing styles
+
+In the AsyncApi component we use pure css styling. Each component inside root component has a unique css class. Each class has form: `asyncapi__{ELEMENT}--{MODIFIER}`, where:
```

Comment:
maybe `each html tag has unique css class`? from css perspective there are html tags/elements, not components